packages/web/components/Player/PlayerTile.tsx

```diff
@@ -47,10 +47,13 @@ export const PlayerTile: React.FC<Props> = ({
   index,
 }) => {
   const description = getPlayerDescription(player);
+  /** 
   const displayDescription =
     typeof description === 'string' && description.length > MAX_BIO_LENGTH
       ? `${description?.substring(0, MAX_BIO_LENGTH - 9)}…`
       : description;
+  */
+  const displayDescription = typeof description === 'string' ? description : '';
```

Comment:
Since `line-clamp` is being used, there is only a single `description`, and `displayDescription` isn't needed.

```suggestion
```

Reply:
✔